src/components/journeys/JourneyInstancesDataTable/getTagColumns.tsx

```diff
@@ -21,7 +21,7 @@ const getTagColumns = (tagMetadata: TagMetadata): GridColDef[] => {
     renderCell: (params) => {
       if (group) {
         return getGroupTags(params.row, group.id).map((tag) => (
-          <TagChip key={tag.id} tag={tag as ZetkinTag} />
```

Comment:
Unrelated but you don't need to typecast this, `getGroupTags` returns a list of tags! woo

Reply:
But i would just merge, it's typecasting from and to the same type. can deal with later.

Reply:
Ah, this is probably since when the tags on journey instances were only partial tags (`Pick<…>`). But yeah I agree about dealing with it later. 😊